.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+end_of_line = lf
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
+
+[*.{js,json}]
+indent_size = 2
+indent_style = space
+
+[*.md]
+insert_final_newline = false
```

Comment:
@rimiti  I think we should still insert final newlines even for md files.

Reply:
Hum, ok! I fix it